Week-6/W6Exercise6.1/Index.js

```diff
@@ -0,0 +1,22 @@
+const maxSumContiguousSubArray = (_arr) => {
+    let maxSoFar = Number.MIN_VALUE, maxEndingHere = 0, start = 0, end = 0, temp = 0;
+    for (let i = 0; i < _arr.length; i++) {
```

Comment:
Explain this loop in laymen terms.

Reply:
In loop

- Adding the elements
- Checking if the added value is greater than maxSoFar which is min value present in number, assuming if the array could also consist of only negative values.
- If it is greater then, maxSofar is assigned value of total added till now.
- start & end, variables are used for returning the start and end of the sub array
- checking if total added so far is still less than zero, reassigning it to zero, and temp value to i+1